src/Data/Yaml/Extra.hs

```diff
@@ -1,17 +1,21 @@
+{-# LANGUAGE ScopedTypeVariables #-}
 -- | Wrappers for Yaml functions to workaround
 -- https://github.com/commercialhaskell/stack/issues/2491.
 -- Import Data.Yaml.Extra in place of Data.Yaml to use this workaround.
 -- Beware these functions construct/deconstruct the entire file at once!
 module Data.Yaml.Extra (decodeFileEither, encodeFile, module Data.Yaml) where
 
+import           Control.Exception
```

Comment:
Since he seems busy, I'll trust my (more educated) judgement, assume this is fine, and merge this. Details below but feel free to skip.

Based on https://github.com/commercialhaskell/haskelldocumentation/blob/master/content/exceptions-best-practices.md, having read some more docs (including for `MonadCatch`, which doesn't apply here), having read the papers on Haskell exceptions (including async ones, also not applying here*), I think this is fine.

*We risk catching any async `IOException`, and I understand that's bad. But `IOException` shouldn't be used for asynchronous exceptions, and even `safe-exceptions` won't help there.
https://www.fpcomplete.com/blog/2016/06/announce-safe-exceptions